src/backend/blockchain/src/blockchain.py

```diff
@@ -3,9 +3,11 @@
 from .transaction import Transaction
 from .exceptions import WalletException
 
+
 class Blockchain:
 
     difficulty = 4
+    COINBASE_AMOUNT = 10
```

Comment:
Is this coinbase_amount used anywhere?

Reply:
It's used on line 52 in `blockchain/src/routes.py`